src/integration-test/java/org/ow2/proactive/catalog/service/CatalogObjectServiceIntegrationTest.java

```diff
@@ -134,8 +134,19 @@ public void testListCatalogObjectsByKindInBucket() {
                                                                                                    "object");
         assertThat(catalogObjects).hasSize(2);
 
-        catalogObjects = catalogObjectService.listCatalogObjectsByKind(bucket.getName(), "workflow");
+        catalogObjectService.createCatalogObject(bucket.getName(),
+                                                 "catalog4",
```

Comment:
"catalog4" is normal or you meant "catalog"?

Reply:
I just follow the logic of creating previous objects in this test with name catalog1, catalog2